src/Markdown.MAML/Transformer/MamlModelMerger.cs

```diff
@@ -286,42 +284,6 @@ private void MergeParameters(MamlCommand result, MamlCommand metadataModel, Maml
                 Report($"    Exception parameter merge: \r\n{ex.Message}\r\n");
                 _cmdletUpdated = true;
             }
-
-
-        }
-
-        /// <summary>
-        /// Compares parameters
-        /// </summary>
-        private string metadataStringCompare(string metadataContent, string stringContent, string moduleName, string paramName, string contentItemName)
-        {
-            var pretifiedStringContent = stringContent == null ? "" : Pretify(stringContent).TrimEnd(' ');
-            var pretifiedMetadataContent = metadataContent == null ? "" : Pretify(metadataContent).TrimEnd(' ');
-
-            return stringContent;
-        }
-
-        /// <summary>
-        /// Cleans the extra \r\n and inserts a tab at the beginning of new lines, mid paragraphs
-        /// </summary>
-        private static string Pretify(string multiLineText)
-        {
-            if(string.IsNullOrEmpty(multiLineText))
-            {
-                multiLineText = "";
-            }
-            return Regex.Replace(multiLineText, "(\r\n)+", "\r\n    ");
-        }
-
-        /// <summary>
-        /// Compares Cmdlet values: Synopsis, Description, and Notes. Preserves the content from the string model (old) or the metadata model (new).
-        /// </summary>
-        private string metadataStringCompare(string metadataContent, string stringContent, string moduleName, string contentItemName)
-        {
-            var metadataContentPretified = (metadataContent == null ? "" : Pretify(metadataContent).TrimEnd(' '));
-            var stringContentPretified = (stringContent == null ? "" : Pretify(stringContent).TrimEnd(' '));
-
-            return stringContent;
```

Comment:
I'm assuming lots of emoticons is good :) So yeah the code block doesn't currently do anything meaningful and just needed to be cleaned up so I removed it instead of rewriting it to be compatible with the SectionBody class implementation.